generator/test-spread-attributes/template.templ

```diff
@@ -0,0 +1,7 @@
+package testspreadattributes
+
+templ BasicTemplate(spread templ.Attributes) {
```

Comment:
Could be worth a test that combines the spread with a conditional, just to prove that it works.